### PR TITLE
[Flutter Parent][MBL-13912] Calendar week/month

### PR DESCRIPTION
**Pull request by JordanMarshall**

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by JordanMarshall** [comment]
@StephenBrough Good luck

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1024300&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1024301&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1024302&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/flutter_parent/test/screens/calendar/calendar_widget/calendar_day_test.dart [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+      selectedDay: selectedDate,
+      onDaySelected: (_) {},
+      eventCount: CalendarEventCount(),
+    )));
+    await tester.pump();
+
+    expect(find.bySemanticsLabel('Saturday, January 1'), findsOneWidget);
+  });
+
+  testWidgetsWithAccessibilityChecks('Sets a11y description', (tester) async {
```
[the remark this answers is not in the document]
Comment (JordanMarshall) [reply apps/flutter_parent/test/screens/calendar/calendar_widget/calendar_day_test.dart]: But of course. You know what they say about tricky code - "Won't play nice? Test it twice!"

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1024341&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Pull request merged**